manifests/vrrp/unicast_peer.pp

```diff
@@ -0,0 +1,23 @@
+# == Define: keepalived::vrrp::unicast_peer
+#
+# === Parameters:
+#
+# $instance::   Name of vrrp instance this peer belongs to
+#
+# $ip_address:: IP address of unicast peer
+#               Default: $name
+#
+define keepalived::vrrp::unicast_peer (
+  String $instance,
+  Stdlib::IP::Address $ip_address = $name,
+) {
+  assert_private()
```

Comment:
I never saw assert_private() in a defined resource. does that actually work and limit the scope of the resource to this module?

Reply:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#assert_private says "class or definition" so I suppose it must. I did add `@api private` to the documentation, per https://puppet.com/docs/puppet/5.5/puppet_strings.html#available-strings-tags